web/app/themes/npm/package.json

```diff
@@ -29,16 +29,15 @@
     },
     "dependencies": {
         "bootstrap": "^3.4.1",
-        "bootstrap-dropdown-replacement": "^1.0.7",
         "jquery": "^3.4.0"
     },
     "devDependencies": {
-        "cross-env": "^5.2.0",
+        "cross-env": "^7.0.2",
```

Comment:
Hi. It might worth noting that 'cross-env' isn't used in this site.

Reply:
Ok thanks :) I'll remove it, test it, push up that commit then merge

Reply:
Now removed, so I'll merge